src/RuleSet.php

```diff
@@ -241,7 +241,28 @@ final class RuleSet implements RuleSetInterface
             '@Symfony' => true,
             'align_multiline_comment' => true,
             'array_indentation' => true,
-            'blank_line_before_statement' => true,
+            'blank_line_before_statement' => [
+                'statements' => [
+                    'break',
+                    'case',
+                    'continue',
+                    'declare',
+                    'default',
+                    'die',
+                    'exit',
+                    'goto',
+                    'include',
+                    'include_once',
+                    'require',
+                    'require_once',
+                    'return',
+                    'switch',
+                    'throw',
+                    'try',
+                    'yield',
```

Comment:
I would not add `yield` here: it can be convenient to write concise PHPUnit test case providers. The rule will add a lot of blank lines there (as shown in this PR) which reduces lisibility IMO.